Userland/Libraries/LibAudio/FlacLoader.cpp

```diff
@@ -784,6 +789,11 @@ ErrorOr<Vector<i32>, LoaderError> FlacLoaderPlugin::decode_custom_lpc(FlacSubfra
 // Decode a subframe encoded with one of the fixed linear predictor codings
 ErrorOr<Vector<i32>, LoaderError> FlacLoaderPlugin::decode_fixed_lpc(FlacSubframeHeader& subframe, BigEndianInputBitStream& bit_input)
 {
+    // LPC must provide at least as many samples as its order.
+    if (subframe.order > m_current_frame->sample_count) {
+        return LoaderError { LoaderError::Category::Format, static_cast<size_t>(m_current_sample_or_frame), "Too small frame for LPC order" };
+    }
```

Comment:
You can drop the braces for these single-line blocks in this commit (as per the other commits)